dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/JaxRsMonitorFilter.java

```diff
@@ -189,20 +301,33 @@ public void filter(ContainerRequestContext reqCtx, ContainerResponseContext resp
          * attribute as "RESTFUL.HTTP.ROUTE"
          */
         if (resourceClass != null && resourceMethod != null) {
-            String route;
+            String route = getRoute(resourceClass, resourceMethod);
```

Comment:
Similarly can we use the getRoute method in telemetry to make the code more common?